docker/runtime/python-2.7/Dockerfile

```diff
@@ -1,9 +1,15 @@
 FROM python:2.7.11-alpine
 
-RUN apk add --no-cache python py-pip git
-RUN pip install --upgrade pip
+RUN apk add --no-cache python py-pip uwsgi-python
+RUN pip install -U pip
 RUN pip install bottle
 
-ADD kubeless.py /
+ADD uwsgi.ini kubeless.py /
 
-CMD ["python", "/kubeless.py"]
+ENV \
+ UWSGI_HTTP_SOCKET=:8080 \
```

Comment:
did you check this was serving on 0.0.0.0, sometimes I have seen issues where the default serves on localhost inside the container and we cannot reach the app from outside the container.